### PR TITLE
Fixes #6211 contact custom label name instead of phone number when av…

### LINKED ISSUE #6211: [Feature Request] Add Contact's custom string to Signal app
My contacts are organized by client, and the phone numbers listed as 'so & so's cell'.

Is it possible to add the 'custom' string under the Name of the Contact in Signal? That way I can easily tell who I'm chatting with.

I don't know a better way to organize lots of numbers with Google. It's been rather clunky usually, but this way I can quickly find people.

### PR DESCRIPTION
**Pull request by Bilb**
…ailable

### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Nexus 5, Android 7.1.1
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description

Enable a feature asked by some people, and that I find useful too. Shows the "custom label" set in the contact field as a subtitle in the conversation view, if available. This is an easy one, before going deeper in bugs resolution/features addition.

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ConversationTitleView.java [hunk 1]
```diff
@@ -67,7 +67,10 @@ private void setRecipientTitle(Recipient recipient) {
         this.subtitle.setVisibility(View.GONE);
       } else {
         this.title.setText(recipient.getName());
-        this.subtitle.setText(recipient.getNumber());
+        if(recipient.getCustomLabel() != null)
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/ConversationTitleView.java]: "if ("

Comment (moxie0) [inline src/org/thoughtcrime/securesms/ConversationTitleView.java]: please use braces when if conditions span multiple lines

**Comment by Bilb** [comment]
Well, currently I am using the getCustomLabel() method, so I only use the custom label set by the user if defined. It means the alias like "Cellphone, Work, Home, ..." will not be used, unless set as custom label. You can check in the contact app : you can choose "Home, Work, etc" or "custom" and this last one is the one I use. 

In fact, using all available aliases would be nice, but we would need something to show the phone number associated with it. A tap on the alias would be nice to toggle alias/phone number, but currently it is used to show the preferences of conversation activity (already available via the menu button).

What do you think about the toggle label/phone number idea?

File: src/org/thoughtcrime/securesms/ConversationTitleView.java [hunk 2]
```diff
@@ -67,7 +67,14 @@ private void setRecipientTitle(Recipient recipient) {
         this.subtitle.setVisibility(View.GONE);
       } else {
         this.title.setText(recipient.getName());
-        this.subtitle.setText(recipient.getNumber());
+
+        if(recipient.getCustomLabel() != null) {
```
Comment (p4nci) [inline src/org/thoughtcrime/securesms/ConversationTitleView.java]: just few lines above you have an example for if-else: single line else, "if (" with whitespace
[Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)

**Comment by moxie0** [comment]
Thanks @Bilb maybe I'm not sure what the problem we're trying to solve is? In general I'm not opposed to the idea of having a label if we can find a place that doesn't look too busy, but I don't really want to replace the number at this time.  A toggle might be fine, but as you say there's already a tap action there.

**Comment by moxie0** [comment]
in 4.6.0

**Pull request closed without merging**